File: converter-dotnet-core-console-vb/TextFileConversion.vb
Imports System.IO
Imports ceTe.DynamicPDF.Conversion

Public Class TextFileConversion
    Public Shared Sub Run(outputPdf As String)

        Dim options As TextConversionOptions = New TextConversionOptions(PageSize.Legal, PageOrientation.Portrait, 50)
        Dim textConverter As TextConverter = New TextConverter(File.ReadAllText(Program.GetResourcePath("DocumentA.txt")), options)
        textConverter.Convert(Program.GetOutputDocPath(outputPdf))
        Console.WriteLine("TxtFileConversion Example Executed.")

    End Sub
End Class

File: converter-dotnet-core-console-cs/TextFileConversion.cs
using ceTe.DynamicPDF.Conversion;
using System;
using System.Collections.Generic;
using System.IO;
using System.Text;

namespace converter_dotnet_core_console_cs
{
    public class TextFileConversion
    {
        public static void Run(string outputPdf)
        {
            TextConversionOptions options = new TextConversionOptions(PageSize.Legal, PageOrientation.Portrait, 50);

            TextConverter textConverter = new TextConverter(File.ReadAllText(Program.GetResourcePath(@"DocumentA.txt")), options);

            textConverter.Convert(Program.GetOutputDocPath(outputPdf));

            Console.WriteLine("TxtFileConversion Example Executed.");
        }
    }
}
